contracts/makefile

```diff
@@ -22,7 +22,7 @@ lint :; yarn solhint "{scripts,src,test}/**/*.sol"
 
 anvil :; anvil -m 'test test test test test test test test test test test junk'
 
-clear-anvil-deployments :; rm -rf deployments/alpha/base_anvil deployments/base_anvil deployments/river_anvil
+clear-anvil-deployments :; rm -rf deployments/local_multi
```

Comment:
there are two local deployments, `local_multi` and `local_multi_de` (for disable entitlements). 
the plan is to re-write the sdk tests to run on entitled backends, but that's a chore and will happen once the sdk is more mature.

Reply:
Tests cover both paths, so if tests are working it must be doing the setup right, i'm just weary of the hard coded env here.